chainregistry.go

```diff
@@ -10,6 +10,7 @@ import (
 	"sync"
 	"time"
 
+	"decred.org/dcrwallet/wallet"
```

Comment:
We have a proxy now? or just for dcrwallet?

Reply:
Just for dcrwallet at the moment.